### PR TITLE
[ISSUE-637] Support k8s 1.22 version for csi

### LINKED ISSUE #637: Support CSI Baremetal on k8s 1.22
**Is your feature request related to a problem? Please describe.**
Currently CSI Baremetal not works on k8s 1.22. Need to support k8s 1.22 at CSI.

**Describe the solution you'd like**
Need to update sidecar images.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Aber4Nod**
Signed-off-by: Nikita Mikhnenko <nikita_mikhnenko@dell.com>

### Resolves #637 

Updating csi-provisioner version to v3.0.0 to support k8s v1.22.0

## PR checklist
- [x] Add link to the issue
- [x] Choose Project
- [ ] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Provide test details_

### CODE REVIEW

File: variables.mk [hunk 1]
```diff
@@ -22,7 +22,7 @@ TAG              := ${FULL_VERSION}
 BRANCH           := $(shell git rev-parse --abbrev-ref HEAD)
 
 ### third-party components version
-CSI_PROVISIONER_TAG := v1.6.0
```
Comment (mishoyama) [inline variables.mk:25]: Old tag is still in use, right? https://github.com/dell/csi-baremetal-operator/blob/21a6f06652587b1e426ff9094f3f6b1d56882bd3/charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml#L45

Comment (Aber4Nod) [reply variables.mk:25]: Yes, but for out kind version 1.18 it's ok to use provisioner version 3.0.

Comment (mishoyama) [reply variables.mk:25]: Do you mean the provisioner 3.0 will work with k8s 1.18 as well?

Comment (Aber4Nod) [reply variables.mk:25]: Yes, it can. v1 api version of CSINode crd was introduced in k8s 1.17

Comment (mishoyama) [reply variables.mk:25]: In this case why do we need to have k8s version check here - https://github.com/dell/csi-baremetal-operator/blob/master/charts/csi-baremetal-deployment/templates/csi-baremetal_v1_deployment.yaml#L42 and deprecate tag here - https://github.com/dell/csi-baremetal-operator/blob/master/charts/csi-baremetal-deployment/values.yaml#L83 if minimal supported version is 1.18?

Comment (Aber4Nod) [reply variables.mk:25]: If we do not support the earlier versions - than there is no need in it

**Review by mishoyama — APPROVED** [review]

**Review by safronovD — APPROVED** [review]

**Pull request merged**